### DISCUSSION TITLE
App Connection to mssql server problem

### DISCUSSION THREAD

**Discussion by mihanrayan-co** [Budibase/budibase#8757] [Help]
**Hosting**
- Self
  - Method: docker compose
  - Budibase Version: 1.2.52
  - App Version: 1.2.52
**Describe the bug**
I've created a dashboard application that connects to a SQL server running on the internet. first, when I was making this app I used a database backup which was restored on Budibase's local network. during that time it was working fine and the speed was good. now that the app's development has ended and I want to use the real SQL server I've ended up with a very annoying query response delay. Each page takes about 20 seconds to fetch data from the SQL server. the SQL server is completely fine and other applications have connected to it and working properly. this latency not only happens inside the app that I've developed but also through the Budibase App portal itself where it shows database-fetched tables records. I've tried to figure out the problem by following Budibase docker-compose logs after each request to Budibase API, and observing the chrome network tab, ... . not of them doesn't help and right now the app it's completely useless.

**To Reproduce**
1. Go to 'Apps'
2. Click on 'Manage'
3. View app

**Comment by melohagan** [comment]
Hey @mihanrayan-co 

Perhaps a lot of data is being queried through joins - does your app have many relationships setup? If so, would you be able to remove the relationships as a test to see if that could be the cause.
